ui/app/src/services/content.ts

```diff
@@ -395,8 +395,25 @@ function writeContentUrl(qs: object) {
   return `/studio/api/1/services/api/1/content/write-content.json?${qs.toString()}`;
 }
 
-export function updateField(modelId: string, fieldId: string, value: any): Observable<any> {
-  throw new Error('Not implemented.');
+export function updateField(site: string, modelId: string, fieldId: string, value: any): Observable<any> {
+  return getDOM(site, modelId).pipe(
+    switchMap((doc) => {
+      const qs = {
+        site,
+        path: modelId,
+        unlock: 'true',
+        fileName: getInnerHtml(doc.querySelector('file-name'))
+      };
+
+      let fieldNode = doc.querySelector(`:scope > ${fieldId}`);
```

Comment:
fieldNode may not yet exist. Check for null before setting the innerHTML. If null, you need to first append the new child and then you can set the innerHTML. Basically add an if in between the query and the assignment which body creates the element and appends to the doc.